pkg/notify/notify_test.go

```diff
@@ -107,7 +107,7 @@ func (f *fakeJira) Create(issue *jira.Issue) (*jira.Issue, *jira.Response, error
 
 	// Assuming single label.
 	query := fmt.Sprintf(
-		"project=\"%s\" and labels=%q order by resolutiondate desc",
+		"project in('%s') and labels=%q order by resolutiondate desc",
```

Comment:
Why is that change?

Reply:
We must use an "in" JQL expression to compare "project" to a list of projects instead of testing equality with a single project. It works fine if list length is 1.